components/table/index.css

```diff
@@ -10,65 +10,389 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+
+  &.spectrum-Table--sizeS {
+    /* Size and Spacing */
+    --mod-table-min-row-height: var(--mod-table-min-row-height--spacious, var(--spectrum-table-row-height-small-spacious));
+    --mod-table-row-top-to-text: var(--mod-table-row-top-to-text--spacious, var(--spectrum-table-row-top-to-text-small-spacious));
+    --mod-table-row-bottom-to-text: var(--mod-table-row-bottom-to-text--spacious, var(--spectrum-table-row-bottom-to-text-small-spacious));
+
+    /* Row Selection */
+    --mod-table-row-checkbox-block-spacing: var(--mod-table-row-checkbox-block-spacing--spacious, var(--spectrum-table-row-checkbox-to-top-small-spacious));
   }
+
+  &.spectrum-Table--SizeL {
+    /* Size and Spacing */
+    --mod-table-min-row-height: var(--mod-table-min-row-height--spacious, var(--spectrum-table-row-height-large-spacious));
+    --mod-table-row-top-to-text: var(--mod-table-row-top-to-text--spacious, var(--spectrum-table-row-top-to-text-large-spacious));
+    --mod-table-row-bottom-to-text: var(--mod-table-row-bottom-to-text--spacious, var(--spectrum-table-row-bottom-to-text-large-spacious));
+
+    /* Row Selection */
+    --mod-table-row-checkbox-block-spacing: var(--mod-table-row-checkbox-block-spacing--spacious, var(--spectrum-table-row-checkbox-to-top-large-spacious));
+  }
+
+  &.spectrum-Table--SizeXL {
+    /* Size and Spacing */
+    --mod-table-min-row-height: var(--mod-table-min-row-height--spacious, var(--spectrum-table-row-height-extra-large-spacious));
+    --mod-table-row-top-to-text: var(--mod-table-row-top-to-text--spacious, var(--spectrum-table-row-top-to-text-extra-large-spacious));
+    --mod-table-row-bottom-to-text: var(--mod-table-row-bottom-to-text--spacious, var(--spectrum-table-row-bottom-to-text-extra-large-spacious));
+
+    /* Row Selection */
+    --mod-table-row-checkbox-block-spacing: var(--mod-table-row-checkbox-block-spacing--spacious, var(--spectrum-table-row-checkbox-to-top-extra-large-spacious));
+  }
+}
+
+.spectrum-Table--quiet {
+  --mod-table-border-radius: var(--mod-table-border-radius--quiet, 0px);
+  --mod-table-outer-border-inline-width: var(--mod-table-outer-border-inline-width--quiet, 0px);
+  --mod-table-header-background-color: var(--mod-table-header-background-color--quiet, var(--spectrum-transparent-white-100));
+  --mod-table-header-background-color: var(--mod-table-header-background-color--quiet, var(--spectrum-transparent-white-100));
+  --mod-table-row-background-color: var(--mod-table-row-background-color--quiet , var(--spectrum-transparent-white-100));
 }
 
-.spectrum-Table--sizeM {
-  @remapvars {
-    find: /--spectrum-table(.*)-m-/;
-    filter: color;
-    replace: --spectrum-table$1-;
+/********* HIGH CONTRAST *********/
+@media (forced-colors: active) {
+  .spectrum-Table {
+    --highcontrast-table-selected-row-background-color: SelectedItem;
+    --highcontrast-table-focus-indicator-color: Highlight;
+  }
+
```

Comment:
Have we thought about how we should be handling hover states for selected items in WHCM? Was looking at this since it's similar to what I am working on for side nav and noticed that on hover the selected state disappears when looking at WHCM. 


![Screenshot 2023-05-30 at 4 06 17 PM](https://github.com/adobe/spectrum-css/assets/63808889/066a1753-d5fb-4bbb-b9ea-be4e23594023)
(this is on chrome using the dev tools WHCM emulation.

Reply:
look at it on assistive labs too, this is with keyboard focus over row 2
![Screenshot 2023-05-30 at 4 12 26 PM](https://github.com/adobe/spectrum-css/assets/63808889/b26d6389-ab06-4e22-899f-b0f8ddef7f29)

Reply:
Good catch. I'll update the high contrast colors to play well with the new hover and active colors. The selected background color will stay the same blue.

Reply:
@jenndiaz When you get a chance, can you look over this again for high contrast mode? I ended up refactoring a little more than I was expecting. See the commit description for what was changed.

Reply:
Looks great! I like the approach you have and will follow this pattern for Side Nav.